.gitignore

```diff
@@ -37,6 +37,7 @@ pip-delete-this-directory.txt
 # Unit test / coverage reports
 htmlcov/
 .tox/
+report/
```

Comment:
Out of interest, what's this for?

Reply:
It's the pytest-mpl report images which I have locally, but should include as a build artefact